lib/hub/action_request.js

```diff
@@ -10,8 +10,8 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.ActionRequest = exports.ActionDownloadSettings = exports.ActionVisualizationFormatting = exports.ActionFormatting = exports.ActionFormat = exports.ActionType = void 0;
+const node_fetch_1 = require("node-fetch");
```

Comment:
why is this variable named this way?

Reply:
This is out of curiosity, I see it elsewhere like on like 16 but that one makes more sense to avoid using a common word

Reply:
Oh, this is the js file, please ignore

Reply:
js build artifact, nothing we can control